src/components/Parent.vue

```diff
@@ -1,21 +1,26 @@
 <script>
 import axios from "axios";
+import Grid from "./Grid.vue";
 import { saveSelections, getSelections } from "../utilities/indexedDB-helper";
 
 export default {
+  components: { Grid },
   data() {
     return {
-      myJson: {},
+      loaded: false,
+      myJson: [],
       selection: [],
       version: 0,
       changedLocation: "nowhere",
+      showgrid: true,
     };
   },
 
   methods: {
     openIt() {
```

Comment:
I think we should rename this to be `openSecondaryWindow()` or `openSecondarySpreadsheet()`